### PR TITLE
ability to define custom html report file name

### PR DESCRIPTION
**Pull request by github1**
Hey @gearsandcode - this PR handles essentially the same use case as #15 (set own htmlReportFilename), but is a little more flexible because it allows you to provide a template for the file name.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -55,30 +55,30 @@ function onPrepare() {
...
-          resolve();
-        }
-      });
-  });
+  return browser.getProcessedConfig()
+    .then((protractorConfig) => {
+      const capabilities = protractorConfig.capabilities;
+      browserName = capabilities.browserName;
+      if (pluginConfig.htmlReportFilename) {
+        fileName = eval("'use strict'; `" + pluginConfig.htmlReportFilename.replace(/#{/g, '${') + "`");
```
Comment (gearsandcode) [inline index.js:65]: Can this be done without eval()? I feel like this would get the plugin flagged by the likes of a Fortify scan.

Comment (github1) [reply index.js]: That's a good point. I've changed this to use [expression-eval](https://github.com/donmccurdy/expression-eval#readme) - it parses the provided expression and interprets it without resorting to actual eval.

**Comment by github1** [comment]
> Why the package change?

That was inadvertent. I had published a scoped package of my fork and just forgot to change it back.

**Comment by gearsandcode** [comment]
Ugh. I merged this blindly, but looks like I might need to update the documentation. npm start now requires a port and I'm getting an error from Puppeteer. Any ideas?

npm start -p 4200

> protractor-axe-html-report-plugin@1.1.3 start /Users/jesse/Source/protractor-axe-html-report-plugin
> httpster -d testapp/ -p `node -p 'process.env.HTTP_PORT || require("./spec/environment").webServerDefaultPort'` "4200"

/Users/jesse/Source/protractor-axe-html-report-plugin/node_modules/puppeteer/lib/cjs/puppeteer/common/Puppeteer.js:94
    launch(options = {}) {
                   ^

SyntaxError: Unexpected token =
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/jesse/Source/protractor-axe-html-report-plugin/node_modules/puppeteer/lib/cjs/puppeteer/initialize.js:22:24)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)

**Comment by github1** [comment]
What version of node are you using?

**Comment by gearsandcode** [comment]
That'd be 4.4.1 *facepalm*

I need to do more dev work not on my work laptop.

**Pull request merged**